.changeset/bumpy-times-lead.md

```diff
@@ -0,0 +1,6 @@
+---
+'@builder.io/qwik-city': patch
+'@builder.io/qwik': patch
+---
+
+chore: update devDependencies and configurations
```

Comment:
you only updated esbuild and no configs :)

In any case, I don't think this should be in the changelog, it's not something that's visible to the consumers of qwik. So IMHO you should remove this changeset file.